vertical_ngo/view/vertical_ngo.xml

```diff
@@ -0,0 +1,122 @@
+                <field name="module_logistic_budget"
+                       class="oe_inline"/>
+                <label for="module_logistic_budget"/>
+              </div>
+              <div attrs="{'invisible': [('module_logistic_requisition','=',False)]}">
+                <field name="module_logistic_order_donation"
+                       class="oe_inline"/>
+                <label for="module_logistic_order_donation"/>
+              </div>
+              <div attrs="{'invisible': [('module_logistic_requisition','=',False)]}">
+                <field name="module_logistic_requisition_donation"
+                       class="oe_inline"/>
+                <label for="module_logistic_requisition_donation"/>
+              </div>
+              <div attrs="{'invisible': [('module_logistic_requisition','=',False)]}">
+                <field name="module_logistic_requisition_multicurrency"
+                       class="oe_inline"/>
+                <label for="module_logistic_requisition_multicurrency"/>
+              </div>
+              <div attrs="{'invisible': [('module_logistic_requisition','=',False)]}">
+                <field name="module_logistic_order_multicurrency"
+                       class="oe_inline"/>
+                <label for="module_logistic_order_multicurrency"/>
+              </div>
+            </div>
+          </group>
+           <separator string="Purchase Management"/>
+          <group>
+            <label for="id" string="Features"/>
+            <div>
+              <div>
+                <field name="module_ngo_purchase_requisition"
+                       class="oe_inline"/>
+                <label for="module_ngo_purchase_requisition"/>
+              </div>
+              <div>
+                <field name="module_ngo_purchase"
+                       class="oe_inline"/>
+                <label for="module_ngo_purchase"/>
+              </div>
+              <div>
+                <field name="module_framework_agreement_requisition"
+                       class="oe_inline"/>
+                <label for="module_framework_agreement_requisition"/>
+              </div>
+              <div attrs="{'invisible': [('module_logistic_requisition','=',False)]}">
+                <field name="module_framework_agreement_sourcing"
+                       class="oe_inline"/>
+                <label for="module_framework_agreement_sourcing"/>
+              </div>
+            </div>
+          </group>
+         <separator string="Shipment"/>
+          <group>
+            <label for="id" string="Features"/>
+            <div>
+              <div>
+                <field name="module_ngo_shipment_plan"
+                       class="oe_inline"/>
+                <label for="module_ngo_shipment_plan"/>
+              </div>
+<!--
+              <div>
+                <field name=""
+                       class="oe_inline"/>
+                <label for=""/>
+              </div>
+-->
```

Comment:
replying to https://github.com/OCA/vertical-ngo/pull/182/files#r32202746

I meant to remove this commented block I don't see the point of having this "template" here